### ISSUE TITLE
About Sponsorblock settings 

### ISSUE THREAD

**Issue by Mohammed1520** [YT-Advanced/YT-Advanced#118]
Hi sir
I inow that there is an import and export feature for Sponsorblock. This method depends on copying your settings then paste it whenever you need. I inquire if you could make this settings exported as json file like file generated when exporting REX settings. In my opinion exporting and importing Settings as a file is more beneficial than copy and paste

**Comment by Aryan-Chauhan** [comment]
Yes, very useful. Even I was going to request for the same feature.
